src/utils/processDescription/handlers/index.js

```diff
@@ -537,11 +537,14 @@ export const setAccessionHandler /*: Handler */ = handlerConstructor({
     value: (value) => value,
   },
   match: {
-    value(current) {
+    value(current, description) {
       // $FlowFixMe object-this-reference
       const _current = this.cleanUp(current);
-      for (const { re } of setDBs) {
-        if (re.test(_current)) return true;
+      for (const { name, re } of setDBs) {
+        if (re.test(_current)) {
+          description.set.db = name;
```

Comment:
This feels sort of wrong for mutability reasons. I would have to dig into the code a bit more to see how is that we are using the `match` function and see if is there any risk of side effects.